### ISSUE TITLE
genderize error

### ISSUE THREAD

**Issue by ringprince** [ropensci/gender#50]
The `genderize` method does not work for me

```
> gender("Hillary", method = "genderize")
Error: lexical error: invalid char in json text.
                                       <!DOCTYPE html> <html lang="en"
                     (right here) ------^
```

Have I met the free limit?

**Comment by mspittler** [comment]
Actually I face the same error

**Comment by lmullen** [comment]
@ringprince @mspittler This should be fixed in 75ac368a733f18116c0e5e08d44a5e3ed1dc83a4. If you wouldn't mind, please install the dev version with `devtools::install_github("ropensci/gender")` and test it. I will push a bugfix release to CRAN next week.
